packages/trusted-contribution/src/trusted-contribution.ts

```diff
@@ -38,7 +38,7 @@ function isTrustedContribution(author: string): boolean {
 }
 
 export = (app: Application) => {
-  app.on('pull_request', async context => {
+  app.on(['pull_request.opened', 'pull_request.edited'], async context => {
```

Comment:
I think we want `pull_request.synchronized` too which is when code is pushed to the tracking branch.

Reply:
Looking at these docs it is sort of hard to tell: https://developer.github.com/v3/activity/events/types/#pullrequestevent

But I can add it here.